third_party/move/tools/move-package/src/compilation/compiled_package.rs

```diff
@@ -559,7 +559,11 @@ impl CompiledPackage {
                 },
             )
             .collect::<Vec<_>>();
-        for (dep_package_name, _, _, _) in &transitive_dependencies {
+        let mut source_package_map: BTreeMap<String, Symbol> = BTreeMap::new();
```

Comment:
The map is from path to package name, which is used when traversing all_compiled_units to obtain the package name.